test/test_data.py

```diff
@@ -1,13 +1,90 @@
+from typing import Union, Dict, List, Tuple
 import unittest
+
+from rdflib.util import from_n3
+from rdflib import Graph, URIRef, Literal, BNode
+import pytest
+
 from pyontutils.core import OntResIri
 
+Semantic = Union[URIRef, Literal, BNode]
+
 
 class TestCurationExportTtl(unittest.TestCase):
     @classmethod
     def setUpClass(cls):
         cls.ori = OntResIri('https://cassava.ucsd.edu/sparc/exports/curation-export.ttl')
         cls.graph = cls.ori.graph
+        cls.spaql_templates = SparqlQueryTemplates(cls.ori.graph.namespace_manager)
 
-    def test(self):
+    def test_dataset_group(self):
         """ sparql queries here """
-        self.graph
+        query = self.spaql_templates.dataset_group('https://api.blackfynn.io/datasets/N:dataset:c2a014b8-2c15-4269-b10a-3345420e3d56/subjects/53')
```

Comment:
Maybe define the subject id on a separate line since we will probably end up parameterizing this?